internal/test/fixtures/service.yml

```diff
@@ -9,5 +9,5 @@ spec:
     - protocol: TCP
       port: 80
       targetPort: 9376
-  clusterIP: 10.0.171.239
-  type: LoadBalancer
\ No newline at end of file
+  clusterIP: 10.96.0.1
```

Comment:
The service.yml ClusterIP was invalid ("The Service "test-service" is invalid: spec.clusterIP: Invalid value: "10.0.171.239": provided IP is not in the valid range. The range of valid IPs is 10.96.0.0/16").

This doesn't affect tests or anything. I just happened to try to `kubectl apply` the service into a kind cluster and it failed.